wordpress-mobile/WordPress-Android
# Add additional publish flow test

Part of #9296 

Publishes a basic post with additional post settings. Contains changes from #9792 

To test:

* Upload an image to the device
* Run all the Editor tests, they should all pass.

Update release notes:


**NB:** There is an issue due to the tests long clicking on the media button which can cause the tests to fail, right now the only work for this is to run `adb shell settings put secure long_press_timeout 1500`
